tests/clusterintegrationtest/README.md

```diff
@@ -0,0 +1,89 @@
+# Cluster Integration Test
+
+Before running the tests, please install tiup:
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+source .bash_profile
+tiup --version
+```
+
+## Guide: Run tests
+
+```shell
+# cd clusterintegrationtest
+./run_mysql_tester.sh  # mysql-tester test
+
+# vector python testers
+python3 -m pip install uv
+uv venv --python python3.9
+source .venv/bin/activate
+uv pip install -r requirements.txt
```

Comment:
It is recommended to generate pyproject.toml anduv.lock using uv init. Then `uv run <command>` to start python script test, uv will verify that the lockfile is up-to-date with the pyproject.toml, and that the environment is up-to-date with the lockfile